src/SFML/Modules/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+cmake_minimum_required(VERSION 3.28)
+
+# System module is always required
+set(SFML_MODULES
+    Config.cppm
+    System.cppm
+)
+
+# Add Window module files if enabled
+if(SFML_BUILD_WINDOW)
+    list(APPEND SFML_MODULES
+        Window.cppm
+    )
+endif()
+
+# Add Graphics module files if enabled
+if(SFML_BUILD_GRAPHICS)
+    list(APPEND SFML_MODULES
+        Graphics.cppm
+    )
+endif()
+
+# Add Audio module files if enabled
+if(SFML_BUILD_AUDIO)
+    list(APPEND SFML_MODULES
+        Audio.cppm
+    )
+endif()
+
+# Add Network module files if enabled
+if(SFML_BUILD_NETWORK)
+    list(APPEND SFML_MODULES
+        Network.cppm
+    )
+endif()
+
+# Define configure_cpp_module_target function if not already defined
+if(NOT COMMAND configure_cpp_module_target)
+    function(configure_cpp_module_target target)
+        target_sources(${target} PUBLIC FILE_SET CXX_MODULES FILES ${SFML_MODULES})
+    endfunction()
+endif()
```

Comment:
This function is used only once. Let's remplace its usage by its body and not define the function.